language-server/modules/langserver-core/src/test/resources/completion/new/typedesc_context/config/error_typedesc1.json

```diff
@@ -5,49 +5,6 @@
   },
   "source": "typedesc_context/source/error_typedesc1.bal",
   "items": [
-    {
```

Comment:
@nadeeshaan Need help with verifying these change to completion tests.

Reply:
We have to rewrite the auto-completion logic with this change

Reply:
Created an issue https://github.com/ballerina-platform/ballerina-lang/issues/29088